doc/samtools.1

```diff
@@ -141,6 +151,77 @@ and indexed input file.
 Options exist to change the output format from SAM to BAM or CRAM, so
 this command also acts as a file format conversion utility.
 
+.TP \"-------- tview
+.B tview
+samtools tview
+.RB [ -p
+.IR chr:pos ]
+.RB [ -s
+.IR STR ]
+.RB [ -d
+.IR display ]
+.RI <in.sorted.bam>
+.RI [ref.fasta]
+
+Text alignment viewer (based on the ncurses library). In the viewer,
+press `?' for help and press `g' to check the alignment start from a
+region in the format like `chr10:10,000,000' or `=10,000,000' when
+viewing the same reference sequence.
+
+.TP \"-------- quickcheck
+.B quickcheck
+samtools quickcheck
+.RI [ options ]
+.IR in.sam | in.bam | in.cram
+[ ... ]
+
+Quickly check that input files appear to be intact. Checks that beginning of the
+file contains a valid header (all formats) containing at least one target
+sequence and then seeks to the end of the file and checks that an end-of-file
+(EOF) is present and intact (BAM only).
+
+Data in the middle of the file is not read since that would be much more time
+consuming, so please note that this command will not detect internal corruption,
+but is useful for testing that files are not truncated before performing more
+intensive tasks on them.
+
+This command will exit with a non-zero exit code if any input files don't have a
+valid header or are missing an EOF block. Otherwise it will exit successfully
+(with a zero exit code).
+
+.TP \"-------- index
+.B index
+samtools index
+.RB [ -bc ]
+.RB [ -m
+.IR INT ]
+.IR aln.bam | aln.cram
+.RI [ out.index ]
+
+Index a coordinate-sorted BAM or CRAM file for fast random access.
+(Note that this does not work with SAM files even if they are bgzip
```

Comment:
This is no longer true.  `samtools index` does work on BGZF-compress SAM, and you need to use it instead of tabix if you want samtools index retrieval to work.
Example:
```
$ ./samtools view -h -o /tmp/samtools.sam.gz test/dat/view.001.sam 
$ cp /tmp/samtools.sam.gz /tmp/tabix.sam.gz
$ ./samtools index /tmp/samtools.sam.gz
$ ../htslib/tabix /tmp/tabix.sam.gz
$ ls /tmp/samtools.sam* /tmp/tabix.sam*
/tmp/samtools.sam.gz	  /tmp/tabix.sam.gz
/tmp/samtools.sam.gz.bai  /tmp/tabix.sam.gz.tbi

$ ./samtools view /tmp/samtools.sam.gz ref2:46-46
ref2_grp3_p002	99	ref2	46	99	15M	=	16	-45	CTGTTTCCTGTGTGA	{{{{{{{{{{{{{{{	RG:Z:grp3	NM:i:0	MD:Z:15

$ ./samtools view /tmp/tabix.sam.gz ref2:46-46
[E::idx_find_and_load] Could not retrieve index file for '/tmp/tabix.sam.gz'
[main_samview] random alignment retrieval only works for indexed BAM or CRAM files.
```

Possibly we should add another commit to correct this?